content/guides/tasks/assignments/change-state.md

```diff
@@ -0,0 +1,17 @@
+---
+rank: 7
+related_endpoints:
+  - put_task_assignments_id
+related_guides: []
+required_guides: []
+alias_paths: []
+---
+
+# Change the state of a task assignment
+
+To update a task assignment in Box you will need to call the
+[`PUT /tasks/:task_id/assignments`](e://put_task_assignments_id) API and
+include a `resolution_state`, such as `completed`, `incomplete`,
+`approved`, or `rejected`.
```

Comment:
Do you think we need to document the meaning of these states somewhere? Right now it's hard to grasp what each means. Maybe we could cover this in the index page for tasks?

Reply:
Added a message explaining resolution states, but this guide is slightly ahead of the current docs until the OAS PR is merged

Reply:
Sure, we can merge these 2 in tandem.